tailor_image/create_image.py

```diff
@@ -36,22 +36,39 @@ def create_image(name: str, distribution: str, apt_repo: str, release_track: str
     recipe = yaml.safe_load((rosdistro_path / 'config/images.yaml').open())['images']
     distro = recipe[name]['distro']
     build_type = recipe[name]['build_type']
-    package = recipe[name]['package']
-    provision_file = recipe[name]['provision_file']
     env = source_file(f'{os.environ["BUNDLE_ROOT"]}/{distro}/setup.bash')
-    template_path = find_package(package, f'image_recipes/{name}/{name}.json', env)
     today = datetime.date.today().strftime('%Y%m%d')
     extra_vars = []  # type: List[Any]
 
+    try:
+        package = recipe[name]['package']
+        provision_file = recipe[name]['provision_file']
+        ansible_path = find_package(package, 'ansible.cfg', env)
```

Comment:
can you set `env['ANSIBLE_CONFIG'] = ansible_path` immediately after this, instead of the locals hack